docs/uninstall_and_downgrade.md

```diff
@@ -0,0 +1,244 @@
+   Deleted sat-2.2.10 from product catalog.
+   ```
+
+## Activate: Switch Between SAT Versions
+
+This procedure can be used to downgrade the active version of SAT.
+
+### Prerequisites
+
+- Only versions 2.2 or newer of SAT can be activated. Older versions must be activated manually.
+- CSM version 1.2 or newer must be installed, so that the `prodmgr` command is available.
+
+### Procedure
+
+1. Use `sat showrev` to list versions of SAT.
+
+   ```screen
+   ncn-m001# sat showrev --products --filter product_name=sat
+   ###############################################################################
+   Product Revision Information
+   ###############################################################################
+   +--------------+-----------------+--------+--------------------+-----------------------+
+   | product_name | product_version | active | images             | image_recipes         |
+   +--------------+-----------------+--------+--------------------+-----------------------+
+   | sat          | 2.3.3           | True   | -                  | -                     |
+   | sat          | 2.2.10          | False  | -                  | -                     |
+   +--------------+-----------------+--------+--------------------+-----------------------+
+   ```
+
+1. Use `prodmgr` to activate a different version of SAT.
+
+   This command will do three things:
+
+   - For all hosted-type package repositories associated with this version of SAT, set them as the sole member
+     of their corresponding group-type repository. For example, activating SAT version `2.2.10`
+     sets the repository `sat-2.2.10-sle-15sp2` as the only member of the `sat-sle-15sp2` group.
+   - Set the version `2.2.10` as active within the product catalog, so that it appears active in the output of
+     `sat showrev`.
+   - Ensure that the SAT CFS configuration content exists as a layer in all CFS configurations that are
+     associated with NCNs with the role "Management" and subrole "Master" (for example, the CFS configuration
+     `management-23.03`). Specifically, it will ensure that the layer refers to the version of SAT CFS
+     configuration content associated with the version of SAT being activated.
+
+   ```screen
+   ncn-m001# prodmgr activate sat 2.2.10
+   Repository sat-2.2.10-sle-15sp2 is now the default in sat-sle-15sp2.
+   Set sat-2.2.10 as active in product catalog.
+   Updated CFS configurations: [management-23.03]
+   ```
+
+1. Verify that the chosen version is marked as active.
+
+   ```screen
+   ncn-m001# sat showrev --products --filter product_name=sat
+   ###############################################################################
+   Product Revision Information
+   ###############################################################################
+   +--------------+-----------------+--------+--------------------+-----------------------+
+   | product_name | product_version | active | images             | image_recipes         |
```

Comment:
The `active` column is being removed in Alice.

Reply:
I created a separate ticket to work on updating this information: [STP-3520](https://jira-pro.its.hpecorp.net:8443/browse/STP-3520)